app/src/androidTest/java/com/github/mateo762/myapplication/coaching/OffersFragmentTest.kt

```diff
@@ -0,0 +1,90 @@
+package com.github.mateo762.myapplication.coaching
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.mateo762.myapplication.coaching.fragments.RequestsFragment
+import com.github.mateo762.myapplication.models.HabitEntity
+import com.github.mateo762.myapplication.models.UserEntity
+import com.github.mateo762.myapplication.ui.coaching.OffersScreen
+import com.github.mateo762.myapplication.ui.coaching.RequestsScreen
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseReference
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
+import org.mockito.MockitoAnnotations
+import java.time.DayOfWeek
+
+//@RunWith(AndroidJUnit4::class)
+class OffersFragmentTest {
```

Comment:
Commented it out for now as I wasn't able to run this (and I think it failed the tests because the class was empty without tests. Will continue to try writing tests and then uncomment